configure.d/1_module_mutex.conf

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# Copyright(c) 2022 Intel Corporation
```

Comment:
You may want to add Huawei copyright entry to the header in files that you added/modified.